File: CoreJava/collection-app/src/com/techlabs/collections/LineItem.java
package com.techlabs.collections;

public class LineItem {
	private int id;
	private String product;
	private int quantity;
	private double unitprice;

	public LineItem(int id, String product, int quantity, double unitprice) {
		super();
		this.id = id;
		this.product = product;
		this.quantity = quantity;
		this.unitprice = unitprice;
	}

	public int getId() {
		return id;
	}

	public String getProduct() {
		return product;
	}

	public int getQuantity() {
		return quantity;
	}

	public double getUnitprice() {
		return unitprice;
	}

	public double calcTotal() {
		return getQuantity() * getUnitprice();
	}
}

File: DotNET/C#/LineItemApp/LineItemApp/LineItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace LineItemApp
{
    class LineItem
    {
        private int _id;
        private String _product;
        private int _quantity;
        private double _unitprice;

        public LineItem(int id, String product, int quantity, double unitprice)
        {
            _id = id;
            _product = product;
            _quantity = quantity;
            _unitprice = unitprice;
        }

        public int Id
        {
            get
            {
                return _id;
            }
        }

        public String Product
        {
            get
            {
                return _product;
            }
        }

        public int Quantity
        {
            get
            {
                return _quantity;
            }
        }

        public double UnitPrice
        {
            get
            {
                return _unitprice;
            }
        }

        public double CalculateTotal()
        {
            return Quantity * UnitPrice;
        }

        public override string ToString()
        {
            return Id + " " + Product + " " + Quantity + " " + UnitPrice;
        }
    }
}
